src/com/projectkorra/projectkorra/ability/ElementalAbility.java

```diff
@@ -45,8 +54,20 @@ public boolean isTransparent(final Block block) {
 		return isTransparent(this.player, this.getName(), block);
 	}
 
-	public List<String> getEarthbendableBlocks() {
-		return getConfig().getStringList("Properties.Earth.EarthBlocks");
+	public static Set<String> getConfiguredTagMaterials(Set<String> materials) {
+		ListIterator<String> iterator = new ArrayList<String>(materials).listIterator();
+		Set<String> tagMaterials = new HashSet<String>();
+		while (iterator.hasNext()) {
+			String tag = iterator.next();
+			if (tag.startsWith("#")) {
+				tagMaterials.addAll(GeneralMethods.tagToMaterialList(tag));
```

Comment:
I am still pretty sure we will want to remove the tag string from the `Set<String>` parameter inside of this if statement. Again, I worry that someone will think the source lists contain only material strings and will run into issues attempting to convert them. To illustrate what I mean:

Let's say `materials` has the following values: `{"#ice", "snow_block"}`
After "#ice" gets processed, `materials` would no longer contain the tag and would only be the following: `{"ice", "packed_ice", "blue_ice", "snow"}`

Reply:
Makes perfect sense, I'll commit a couple more changes in a few minutes so that tags will be trimmed from the material lists after setting up.